minio/scripts/file-operations.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -e
+
+# Executable that waits for a port at an address to be open.
+# For further details go to: https://github.com/ufoscout/docker-compose-wait
+/wait
+
+echo "Add s3 location alias called storage to minio config"
+mc config host add storage http://minio:9000 longkey verysecretkey
+
+echo "Create private bucket (private by default)"
+mc mb storage/private-bucket
```

Comment:
weren't the operation supposed to use the `aws` cli? There would be some that are out of scope, but it demonstrates S3 compatibility

Reply:
(unless `mc` could be pointed to a real S3 bucket for comparison)

Reply:
It's not mentioned in the ticket but shouldn't be an issue as the commands are very similar. I'll swap `mc` for `awscli`.

Reply:
updated. Please review.